store-implementation/accumulo-store/src/test/resources/schemaWithVisibilities/types.json

```diff
@@ -31,6 +31,9 @@
       "class": "java.lang.String",
       "aggregateFunction": {
         "class": "uk.gov.gchq.koryphe.impl.binaryoperator.StringConcat"
+      },
+      "serialiser": {
+        "class": "uk.gov.gchq.gaffer.serialisation.implementation.StringSerialiser"
```

Comment:
Do you need two different Json files? is this being used by the mentioned parents test?

Reply:
The parent test uses `store-implementation/accumulo-store/src/test/resources/schema/`, this is a different schema but it was throwing an error as the visibility type didn't have a Serialiser assigned. The only other test that I can see that uses this schema is `InputFormatTest` which doesn't actually add any elements, hence why this was probably missed.